sass/parts/_posts_list.scss

```diff
@@ -66,14 +76,19 @@
         flex-direction: column;
         align-items: flex-start;
         padding: 2rem 0;
-    }
 
-    .date {
-        margin-bottom: 0;
-        width: 100%;
-    }
+        .bloglist-meta {
```

Comment:
had to bring them up inside `.bloglist-tags` so that they are specific enough to override the other rules.